07/ID_tryExcept

```diff
@@ -0,0 +1,53 @@
+# Input ID number in Czech Republic
+# write ouf corect/incorect ID
+
+
+# MONTHS 30 DAYS
+months_30 = [4, 6, 9, 11]
+
+# MONTHS 31 DAYS
+months_31 = [1, 3, 5, 7, 8, 10, 12]
+
+# YEARS
+years = list(range(86, 100)) + list(range(00, 23))
+ID_only_numbers = []
+
+while True:
+
+    ID = input('Napiš rodné číslo: ')
+    if ID[6] == "/":
+        try:
+            a = int(ID[:6])
+            b = int(ID[7:])
+        except ValueError:
+            print("špatný počet čísel")
+            continue
+    else:
+        print("chybí lomítko")
```

Comment:
Sem ti to spadne ak tam lomítko bude, ale bude na zlej pozícii, preto by som hlášku upravila - chybí nebo je na zlé pozici.